sygnal.yaml.sample

```diff
@@ -249,3 +249,11 @@ apps:
   #  # Defaults to 512 if unset.
   #  #
   #  #inflight_request_limit: 512
+  #
+  #  # This allows you to specify additional options to send to Firebase.
```

Comment:
You may want to be explicit that admins can specify *any* option from the linked URL here, not just those that are listed below (which is what I initially thought).